Demos/Blazorise.Demo/App.razor

```diff
@@ -14,7 +14,7 @@
     <MessageAlert />
     <NotificationAlert />
     <PageProgressAlert />
-    <ModalProvider />
+    <ModalProvider UseModalStructure />
```

Comment:
I think `UseModalStructure` should be passed with the options. That way users can have better control over individual uses cases.

Reply:
That's the global options. Isn't it passed on the options?  If it isn't I meant to.

Reply:
Yea it seems like it is.
`protected virtual bool GetUseModalStructure => ModalProviderOptions?.UseModalStructure ?? UseModalStructure;`

Reply:
I missed it, sorry. It's all good.

Reply:
I missed it, sorry. It's all good.